contracts/apps/disputable/IDisputable.sol

```diff
@@ -39,4 +39,6 @@ contract IDisputable is ERC165 {
     function supportsInterface(bytes4 _interfaceId) external pure returns (bool) {
         return _interfaceId == DISPUTABLE_INTERFACE_ID || _interfaceId == ERC165_INTERFACE_ID;
     }
+
+    function appId() public view returns (bytes32);
```

Comment:
Maybe we can leverage the new breaking version we are working on to make these `external`

Reply:
What’s the benefit?

Reply:
To avoid having the compiler warning:
> Warning: Functions in interfaces should be declared external.

Reply:
not sure to be honest, however this is a minor detail, not a blocker at all